git/fetch.go

```diff
@@ -0,0 +1,64 @@
+package git
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+type FetchOptions struct {
+}
+
+func FetchRepository(c *Client, opts FetchOptions, repository string) error {
```

Comment:
Why isn't this just Fetch to mirror other commands where it has the same name as the command?

Reply:
I was thinking that when we support more flavours of fetch that there might be different arguments to the function such as the group, or even fetch with no repository at all. There's one variant that allows you to specify more than one repository. Some of those don't really make sense in the options. It's a little unfortunate that go doesn't support function overloading.

I will rename this back to match the rest of the commands and defer some of the decision making until later.

Reply:
We can always refactor later (in fact,  we'll almost certainly need to once fetch-pack is implemented)